pyrelational/model_managers/lightning_model_manager.py

```diff
@@ -60,6 +61,10 @@ def init_trainer(self) -> Tuple[Trainer, ModelCheckpoint]:
         config = self.trainer_config
         config = _add_pyl_trainer_defaults(config)
         callbacks: List[Callback] = []
+        
+        if config["logger"] is None:
```

Comment:
I think default should remain with no logger, at the very least there should be an option to not have a logger.

Reply:
Hi, Thomas. I was trying to match the [Pytorch Lightning API ](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.trainer.trainer.Trainer.html#lightning.pytorch.trainer.trainer.Trainer ) which sets as `None` and uses default behavior of `CSVLogger`. To have no logger - consistent with the lightning API - you'd set it as `logger=False`. So we currently support all this functionality and we match the defaults of PyTorch Lightning. If you want to change it however - we can!

Reply:
I see, but wouldn't that mean that we don't need this since by default `None` would get converted as CSVLogger anyway by the Trainer?